contact.md

```diff
@@ -34,4 +34,10 @@ For general inquiries, please contact the project coordinator:
 	</div>
 </div>
 
-(TODO): partners contacts
+<h3>Partners contacts</h3>
+<ul>
+<li><a target="_blank" href="https://www.its.uni-luebeck.de/kontakt/adresse.html">University of Lübeck, Institute of IT security (ITS)</a></li>
+<li><a target="_blank" href="https://www.ifis.uni-luebeck.de/index.php?id=impressum">University of Lübeck, Institute of Information Systems (IFIS)</a></li>
+<li><a target="_blank" href="https://www.lufthansa-industry-solutions.com/de-de/kontakt">Lufthansa Industry Solutions</a></li>
+<li><a target="_blank" href="https://www.netuse.de/kontakt/">NetUSE AG</a></li>
+</ul>
```

Comment:
Should we put here the names of representatives for each partner?

Reply:
Probably makes sense when we're only linking to general contact forms, but about personal info we should always ask the persons, so let's do that when we talk to each of them next?

Reply:
Yes, @onetwo  is it ok to put your representative here?